sig-api-machinery/README.md

```diff
@@ -70,6 +70,9 @@ The following subprojects are owned by sig-api-machinery:
     - https://raw.githubusercontent.com/kubernetes-sigs/controller-runtime/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes-sigs/controller-tools/master/OWNERS
     - https://raw.githubusercontent.com/kubernetes-sigs/kubebuilder/master/OWNERS
+- **server-upgrade-downgrade**
```

Comment:
I think this is (going to be) a control plane feature--there's a section for that on line 42.

Reply:
> I think this is (going to be) a control plane feature--there's a section for that on line 42.

I must need to re-read.  I remember talking about creating this as a separate tool.  It allowed evolution outside of kube releases and was a clear indication that it was indeed separate from core.

Reply:
> I think this is (going to be) a control plane feature--there's a section for that on line 42.
> 
> I must need to re-read. I remember talking about creating this as a separate tool. It allowed evolution outside of kube releases and was a clear indication that it was indeed separate from core.

Or do you just mean, "yes it will be a separate repo, but it falls into an existing bucket and we can link the repo in the existing section"?

Reply:
> it will be a separate repo, but it falls into an existing bucket and we can link the repo in the existing section

Exactly. It will be a control plane feature when it grows up, no matter how it is invoked or where the code lives.